src/main/java/racingcar/validator/Validator.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.validator;
+
+import java.util.Arrays;
+
+public class Validator {
+
+	private static final String EMPTY_CAR_NAME_ERROR_MESSAGE = "자동차의 이름은 1글자 이상이어야 합니다.";
+	private static final String INVALID_CAR_NAME_ERROR_MESSAGE = "자동차의 이름은 5글자 이하여야 합니다.";
+	private static final String CAR_COUNT_LIMIT_ERROR_MESSAGE = "자동차는 2대 이상이어야 합니다";
+	private static final String ATTEMPT_NUMBER_TYPE_ERROR_MESSAGE = "시도 횟수는 숫자여야 합니다.";
+	private static final String ATTEMPT_NUMBER_RANGE_ERROR_MESSAGE = "시도 횟수는 1회 이상이어야 합니다.";
+
+	private static final int NAME_LENGTH_LIMIT = 5;
+	private static final int CAR_COUNT_LIMIT = 2;
+	private static final int MINIMUM_ATTEMPT_NUMBER = 1;
+	private static final String SEPARATOR_OF_CAR_NAME = ",";
+	private static final String INTEGER_REGEX = "-?\\d+";
+
+	public static void ofCarNames(String input) {
+		Arrays.stream(validateCarCount(input))
+			.forEach(Validator::validateCarName);
+	}
+
+	private static void validateCarName(String name) {
+		if (name.isEmpty()) {
+			throw new IllegalArgumentException(EMPTY_CAR_NAME_ERROR_MESSAGE);
+		}
+		if (name.length() > NAME_LENGTH_LIMIT) {
+			throw new IllegalArgumentException(INVALID_CAR_NAME_ERROR_MESSAGE);
+		}
+	}
+
+	private static String[] validateCarCount(String input) {
+		String[] split = input.split(SEPARATOR_OF_CAR_NAME);
+		if (split.length < CAR_COUNT_LIMIT) {
+			throw new IllegalArgumentException(CAR_COUNT_LIMIT_ERROR_MESSAGE);
+		}
+		return split;
+	}
```

Comment:
```suggestion
	public static void ofCarNames(String input) {
		Arrays.stream(validateCarCount(input))
			.forEach(Validator::validateCarName);
	}

	private static String[] validateCarCount(String input) {
		String[] split = input.split(SEPARATOR_OF_CAR_NAME);
		if (split.length < CAR_COUNT_LIMIT) {
			throw new IllegalArgumentException(CAR_COUNT_LIMIT_ERROR_MESSAGE);
		}
		return split;
	}

	private static void validateCarName(String name) {
		if (name.isEmpty()) {
			throw new IllegalArgumentException(EMPTY_CAR_NAME_ERROR_MESSAGE);
		}
		if (name.length() > NAME_LENGTH_LIMIT) {
			throw new IllegalArgumentException(INVALID_CAR_NAME_ERROR_MESSAGE);
		}
	}

```

위의 제안사항과 같이 먼저 사용되는 private 메서드를 위에 적어준다면 좋습니다. 

그 이유는 우리는 보통 위에서 아래로 글을 읽기에 가까이에 위치하면 이해하기가 더 좋아서에요 😁